### PR TITLE
chore: idempotent container running messages [DET-7555]

### PR DESCRIPTION
**Pull request by stoksc**
## Description
Minor fix for the slurm resource manager.

## Test Plan
- [x] tested manually.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: master/internal/sproto/task.go [hunk 1]
```diff
@@ -237,7 +237,7 @@ func (ev *Event) ToTaskLog() model.TaskLog {
 	case ev.ScheduledEvent != nil:
 		message = fmt.Sprintf("Scheduling %s (id: %s)", description, ev.ParentID)
 	case ev.ContainerStartedEvent != nil:
-		message = fmt.Sprintf("Container for %s has started", description)
+		message = fmt.Sprintf("Resources for %s have started", description)
```
Comment (ioga) [inline master/internal/sproto/task.go:240]: `ev.ResourcesStartedEvent`?

File: master/internal/task/allocation.go [hunk 2]
```diff
@@ -549,6 +549,13 @@ func (a *Allocation) ResourcesStateChanged(
 	case sproto.Starting:
 		a.setMostProgressedModelState(model.AllocationStateStarting)
 	case sproto.Running:
+		if a.resources[msg.ResourcesID].Started != nil {
```
Comment (ioga) [inline master/internal/task/allocation.go:552]: 😔 I wish our entire actor state machine system was designed around a possibility of duplicate messages.

**Review by ioga — APPROVED** [review]

File: master/internal/sproto/task.go (hunk 1 above)
Comment (stoksc) [reply master/internal/sproto/task.go:240]: done

File: master/internal/task/allocation.go (hunk 2 above)
Comment (stoksc) [reply master/internal/task/allocation.go:552]: yeah..

**Pull request merged**